### PR TITLE
Fix wrong log timestamp base on http://stackoverflow.com/a/27865750

### PR DESCRIPTION
**Pull request by hramezani**
fix [issue_1234](https://github.com/benoitc/gunicorn/issues/1234) based on @benoitc [response](https://github.com/benoitc/gunicorn/issues/1234#issuecomment-201217687)

### CODE REVIEW

File: gunicorn/glogging.py [hunk 1]
```diff
@@ -81,7 +102,7 @@
             "generic": {
                 "format": "%(asctime)s [%(process)d] [%(levelname)s] %(message)s",
                 "datefmt": "[%Y-%m-%d %H:%M:%S %z]",
-                "class": "logging.Formatter"
+                "class": "Formatter"
```
Comment (wking) [inline gunicorn/glogging.py]: Doesn't this have to be the full `gunicorn.glogging.Formatter`?

Comment (hramezani) [reply gunicorn/glogging.py]: @wking thanks, I changed it to `gunicorn.glogging.Formatter`

File: setup.py [hunk 2]
```diff
@@ -87,6 +87,9 @@ def run_tests(self):
     zip_safe=False,
     packages=find_packages(exclude=['examples', 'tests']),
     include_package_data=True,
+    install_requires=[
+        'tzlocal',
```
Comment (berkerpeksag) [inline setup.py]: I think our policy is to not have dependencies and I'm not sure it's worth to add `tzlocal`.

Comment (hramezani) [reply setup.py]: @berkerpeksag thanks, I remove `tzlocal`, and convert datetime to local time by pyhton standard library.

Comment (berkerpeksag) [reply setup.py]: I'd say wait for the result of the discussion in #1234.

**Pull request closed without merging**